babel.config.js

```diff
@@ -1,7 +1,6 @@
-module.exports = {
-  retainLines: true,
-  compact: true,
-  comments: false,
-  presets: ["module:metro-react-native-babel-preset"],
-  sourceMaps: false
+module.exports = function(api) {
+  api.cache(true);
+  return {
+    presets: ['babel-preset-expo'],
```

Comment:
We don't use Expo here, so I'm not comfortable with changing the babel preset like this. Is there documentation about it? Otherwise, this feels like a somewhat blocking change.

Reply:
Because React Native doesn't officially support web like Expo does, we made `babel-preset-expo` which extends the Metro preset and adds support for web, this mostly adds tree-shaking and decorator support.

Reply:
just because a library has expo in the name doesn't mean it is only intended for the expo managed workflow :) eg: jest-preset-expo is immensely useful in any react-native app ([info](https://blog.expo.io/testing-universal-react-native-apps-with-jest-and-expo-113b4bf9cc44)) and most of the expo sdk works in any react-native app too ([info](https://blog.expo.io/you-can-now-use-expo-apis-in-any-react-native-app-7c3a93041331)). imo if you're using react-native-web, babel-preset-expo is the best choice